src/toPropertyPath.ts

```diff
@@ -0,0 +1,5 @@
+import { decodePointerFragment } from './decodePointerFragment';
+
+export function toPropertyPath(path: string) {
+  return path.split('/').filter(Boolean).map(decodePointerFragment).join('.');
```

Comment:
Um... this is a bit weird, but I propose we remove `.filter(Boolean)`. Because having an empty string is a valid key value in JSON technically.